rfcs/20210510-binary-size-reduction/README.md

```diff
@@ -0,0 +1,156 @@
+# oneDNN binary size reduction through build system.
+
+## Problem Statement.
+Several oneDNN users due to their usage model specifics would love to see their
+application linked statically with oneDNN (or shared object of the library
+itself) doesn't exceed specific disk space quota. One of such oneDNN users is
```

Comment:
"exceed specific" --> "exceed a specific" 
"One such oneDNN user is the OpenVINO toolkit."